interface/patient_file/history/encounters.php

```diff
@@ -462,14 +462,19 @@ function efmouseover(elem, ptid, encid, formname, formid) {
         $raw_encounter_date = date("Y-m-d", strtotime($result4{"date"}));
         $encounter_date = date("D F jS", strtotime($result4{"date"}));
 
+        //fetch acl for given pc_catid
+        $postCalendarCategoryACO = fetchPostCalendarCategoryACO($result4['pc_catid']);
+        $postCalendarCategoryACO = explode('|',$postCalendarCategoryACO);
+        $authPostCalendarCategory = acl_check($postCalendarCategoryACO[0], $postCalendarCategoryACO[1]);
```

Comment:
updated code so if no aco is set there is authorization (like in form pernissions if no aco is set for a form in registry)